llama_index/llms/openai_utils.py

```diff
@@ -214,7 +214,7 @@ def to_openai_message_dict(
 ) -> ChatCompletionMessageParam:
     """Convert generic message to OpenAI message dict."""
     message_dict = {
-        "role": message.role,
+        "role": message.role.value,
```

Comment:
do we need to change this?

Reply:
thanks @joelbarmettlerUZH it makes sense.
@logan-markewich is it fine for you? want to know your thoughts